### ISSUE TITLE
Google Messages app not appearing in app list.

### ISSUE THREAD

**Issue by Kyoumimasu** [8bitPit/Niagara-Issues#1073]
**Describe the bug**
I have the latest updates for both apps. I switch between Niagra and Nova launchers just to mix things up and have been using Google Messages for my SMS on Nova these past months. I just switched back to Niagra an noticed Messages missing from my favorites and my app list. I have a completely different message apps in my list though, and nothing in my hidden apps. 

But when I go to the Play Store it says I have the Google Messages installed and I can open it from there.. not from my app list.

**To Reproduce**
1. Just having the app with Niagra active...? 

**Smartphone (please complete the following information):**
 - Device: LG G7 ThinQ
 - Android version: 9

**Screenshots**
If applicable, add screenshots to help explain your problem. You can paste them here directly.

I'll do that and edit them back in.

**Comment by Kyoumimasu** [comment]
Okay I don't know what happened but I think I figured it out. Some how the Verizon message app and Google message apps names messed up. My Google Messages app was renamed "Verizon Message". And then the Verizon app was shortened to Message. 

I was taking screen shots and decided to try the "Verizon" one and it looked familiar when it opened.. so I viewed app details and it linked right to Google. How that happened, idk but I'm sorry for writing this all out :( I'll close it.
